### ISSUE TITLE
Allow Repositioning Timers On Linux

### ISSUE THREAD

**Issue by Shadow-XVX** [runelite/runelite#16203]
### Type

Incorrect behavior

### Operating System

Linux

### OS Version

Ubuntu

### Bug description

Repositioning the timers (pots, venge, etc) is supposed to work with alt+click. Alt highlights them in yellow as it is supposed to, but clicking and dragging does nothing.
Please fix this, OR could someone please tell me what line(s) I have to add to ~/.runelite/settings.properties to reposition them from top left to bottom left? Thank you!

Edit: I found out that the code needed for settings.properties is: runelite.InfoBoxOverlay_preferredPosition=BOTTOM_LEFT
Adding that line worked for me, so at this point I am satisfied with the manual workaround.

### RuneLite version

.

**Comment by geheur** [comment]
have you tried changing the `Drag Hotkey` in `Runelite` plugin to something other than alt?
Also, this type of troubleshooting is a better fit for the discord runelite.net/discord.

**Comment by Shadow-XVX** [comment]
> have you tried changing the `Drag Hotkey` in `Runelite` plugin to something other than alt? Also, this type of troubleshooting is a better fit for the discord runelite.net/discord.

Thank you for the reply. I don't see a "Drag Hotkey" plugin available, however I did find out the code needed for the settings.properties file. In case anyone else runs into the same problem, the line I added to ~/.runelite/settings.properties is:
runelite.InfoBoxOverlay_preferredPosition=BOTTOM_LEFT

At this point I no longer care about getting the alt drag issue fixed since manually adding the above line to the settings file worked for me, but I will leave this post up for others' sake unless I am asked to take it down.
